editor/plugins/gizmos/decal_gizmo_plugin.cpp

```diff
@@ -38,6 +38,8 @@
 DecalGizmoPlugin::DecalGizmoPlugin() {
 	Color gizmo_color = EDITOR_DEF("editors/3d_gizmos/gizmo_colors/decal", Color(0.6, 0.5, 1.0));
 
+	create_icon_material("decal_icon", Node3DEditor::get_singleton()->get_editor_theme_icon(SNAME("GizmoDecal")));
```

Comment:
```suggestion
	create_icon_material("decal_icon", EditorNode::get_singleton()->get_editor_theme()->get_icon(SNAME("GizmoDecal"), EditorStringName(EditorIcons)));
```

And others should be similar.

Reply:
Should I also convert all existing icon materials not added by this PR?

Reply:
> Should I also convert all existing icon materials not added by this PR?

No, that's already handled by https://github.com/godotengine/godot/pull/81516

Reply:
Alright, I've converted the icon materials for icons added by this PR.